tests/conftest.py

```diff
@@ -173,6 +178,43 @@ def dataset(request, paths, engine):
     return nvtabular.Dataset(paths, part_mem_fraction=gpu_memory_frac, **kwargs)
 
 
+@pytest.fixture(scope="session")
+def db():
```

Comment:
As a test fixture name - 'db' is pretty general. Maybe we should rename to 'asv_db' or something that indicates this is specific to asv?